### DISCUSSION TITLE
Show current environment next to/instead of manage environments.

### DISCUSSION THREAD

**Discussion by JoakoV3** [Kong/insomnia#7607] [Ideas]
With the insomnia 9.3 release, I can no longer see which enviroment I am in. 

I would like to get something similar as it was before. 

This is now with insomnia 9.3:

![Now](https://github.com/Kong/insomnia/assets/58125197/f12bb7a0-31eb-408b-8550-db8db1efe8f0)

This was before the update: 

![Before](https://github.com/Kong/insomnia/assets/58125197/ad7106da-57b8-42cb-81a8-9fccbecede9d)

See how instead of "Manage Environments", it showed the current environment the user was at. 

I think it is more comfortable to know without clicking in which environment I am at

**Comment by jonataswalker** [comment]
https://github.com/Kong/insomnia/pull/7574 introduced this not helpful change
